sqlalchemy_ibmi/base.py

```diff
@@ -441,15 +442,16 @@ def get_server_version_info(self, dialect):
             return []
 
     def _is_nullable_unique_constraint_supported(self, dialect):
-        """Checks to see if the Db2 version is at least 10.5.
+        """Checks to see if the Db2 version is at least 7.1.
         This is needed for checking if unique constraints with null columns
         are supported.
         """
+        # TODO figure out implementation of this function in DB2DDLCompiler
```

Comment:
I believe from our discussion with Scott and other research this is not supported at all, so this can just return False (or is there a `_servers_supports_...` attribute we can set instead?